lib/step_flow/workflows/workflows.ex

```diff
@@ -31,7 +31,12 @@ defmodule StepFlow.Workflows do
     offset = page * size
 
     query =
-      from(workflow in Workflow)
+      from(
+        workflow in Workflow,
+        join: rights in assoc(workflow, :rights),
+        where: fragment("?::varchar[] && ?::varchar[]", rights.groups, ^user_rights),
+        where: rights.action == "view"
```

Comment:
Do you think it can be more efficient to perform:
```
where: rights.action == "view"
where: fragment("?::varchar[] && ?::varchar[]", rights.groups, ^user_rights),
```
For me comparing array is more complex. So limit the operation by providing a pre-filter is the better.

Reply:
Does it make a difference in term of computation?